content/zh/docs/application-store/built-in-apps/redis-app.md

```diff
@@ -21,21 +21,21 @@ weight: 14291
 
 1. 在 `demo-project` 项目的**概览**页面，点击左上角的**应用商店**。
 
-   ![应用商店](/images/docs/zh-cn/appstore/built-in-apps/redis-app/app-store.PNG)
+   ![应用商店](/images/docs/zh-cn/appstore/built-in-apps/redis-app/app-store.png)
```

Comment:
Check all the images in your local build to see if they are dead.

Reply:
Hum...I checked all the images before making this PR. I'll take a look again.